AirCasting/SDSync/SDSyncController.swift

```diff
@@ -0,0 +1,28 @@
+// Created by Lunar on 17/11/2021.
```

Comment:
How can this still be here?

Reply:
I'm guessing the reference in `.pbxproj` is gone, but the file stayed in the filesystem. Just remove it and it will vanish